### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by ntadimeti**
Updating the dependencies of atacworks to newer versions.

TODO: Test and verify that all tutorials / tests work as expected

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,22 +8,21 @@
...
 
-configargparse~=0.15.2
+configargparse~=1.2.3
 Cython~=0.29.13
-h5py~=2.10.0
-numpy~=1.17.0
-pandas~=1.0.0
+h5py~=3.1.0
+numpy~=1.19.4
+pandas~=1.2.0
```
Comment (ohadmo) [inline requirements.txt:15]: no wheel package for Python3.6, https://pypi.org/project/pandas/1.2.0/#files
@ntadimeti can you also update the Readme file that we now support Python >= 3.7

Comment (ntadimeti) [reply requirements.txt:15]: done!

**Comment by ohadmo** [comment]
rerun tests

**Comment by ohadmo** [comment]
rerun tests

**Review by avantikalal — APPROVED** [review]

**Pull request merged**